src/main/java/logging/Logger.java

```diff
@@ -90,4 +90,45 @@ private synchronized void log(LogLevel level, String text)
         for (String s : lines)
             System.out.println("[" + componentName + "][" + level.name() +  "]: " + s);
     }
+
+    /**
+     * Logs a byte array to verbose output
+     * @param bytes The byte array to log
+     */
+    public synchronized void logVerbose(byte[] bytes)
+    {
+        logVerbose(getByteArrayString(bytes));
+    }
+
+    /**
+     * Logs a byte array to quiet output
+     * @param bytes The byte array to log
+     */
+    public synchronized void logQuiet(byte[] bytes)
+    {
+        logQuiet(getByteArrayString(bytes));
+    }
+
+    /**
+     * @param bytes The byte array to print
+     * @return A comma separated list of all bytes in the array
+     */
+    private synchronized String getByteArrayString(byte[] bytes)
+    {
+        StringBuilder byteStr = new StringBuilder("Byte Array: { ");
+
+        // Add individual bytes to the string
+        for(int i = 0; i < bytes.length; i++) {
+            byte b = bytes[i];
+            byteStr.append(b);
```

Comment:
I like it when the bytes are formatted as 2 digit hex values, what does everyone else think?

```java
String.format("%02x ", b)
```